src/code/templates/fv-egg-sort-game.js

```diff
@@ -104,8 +104,8 @@ let animationEvents = {
       fadeDrakeAway: { id: 2, complete: false, animate: function() {
         const { baskets, scale } = _this.props,
               targetBasket = targetBasketIndex >= 0 ? baskets[targetBasketIndex] : null,
-              leftOffset = 122,
-              topOffset = 204.5;
+              leftOffset = 70.5,
+              topOffset = 77.5;
```

Comment:
These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.